packages/components/package.json

```diff
@@ -84,6 +85,7 @@
 	"scripts": {
 		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
 		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json && copy-assets",
-		"prepack": "yarn run clean && yarn run build"
+		"prepack": "yarn run clean && yarn run build",
+		"storybook": "sb dev"
```

Comment:
This script structure is consistent with the other packages that offer an isolated Storybook, e.g. `launchpad` or `composite-checkout`.